README.md

```diff
@@ -136,20 +136,18 @@ operation you would do in any ruby object.
 
 #### Locations
 
-Fontist lets find font locations from a YAML manifest of the following format:
+Fontist lets find font locations from a manifest of the following format:
```

Comment:
Need to add documentation of `from_yaml` loading mechanism.

Reply:
Added